docs/source/contributing/writing-docs.adoc

```diff
@@ -159,3 +159,10 @@ $ rake build
 ----
 
 If the build completes successfully, the site is available under *_preview/openshift-origin/latest/welcome/index.html_*.
+
+[[building-openshift-docs-automatically]]
+=== Building the OpenShift Documentation Automatically
+
+To automatically get the view of {project} documentation integrated into the OpenShift documentation, trigger the link:https://ci.centos.org/job/minishift-docs/build[minishift docs] job by specifying your repo URL in `REPO` and branch name containing your documentation changes in `BRANCH` parameters respectively. This job will perform all necessary steps including the steps mentioned in xref:manually-building-openshift-docs[Manually Building the OpenShift Documentation].
```

Comment:
One sentence per line, please.

How do I trigger the job? Are the parameters only given through the web UI, or is there a way to provide this information on the command line (or both)? Also, would it be more technically accurate to refer to the "**minishift-docs** project" instead of the "minishift docs job"?

Reply:
@robin-owen , only UI.

Also, referring as `minishift-docs job`. There is no project as such.